### PR TITLE
Make getStorage method to take Entity class as a parameter.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**
We need to make getStorage to take an entity class as a parameter because we need to create a storage with the same columns that are used in further test cases.

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java [hunk 1]
```diff
@@ -70,26 +71,28 @@
...
+        storage = getStorage(Entity.class);
     }
 
     @After
     public void tearDownAggregateStorageTest() {
         close(storage);
     }
 
+
+
```
Comment (dmdashenkov) [inline server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java]: Please eliminate the redundant empty lines.

File: server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java [hunk 2]
```diff
@@ -70,26 +71,28 @@
...
     /**
      * Used to get a storage in tests with different ID types.
      *
      * <p>NOTE: the storage is closed after each test.
      *
+     * @param idClass class of aggregate ID
```
Comment (dmdashenkov) [inline server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java]: Please align the code properly.

File: server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java [hunk 3]
```diff
@@ -70,26 +71,28 @@
...
+     * @param aggregateClass aggregate class
      * @param <I> the type of aggregate IDs
      * @return an empty storage instance
      */
-    protected abstract <I> AggregateStorage<I> getStorage(
-            Class<? extends Aggregate<I,
-                                      ? extends Message,
-                                      ? extends ValidatingBuilder<?, ?>>> aggregateClass);
+
+    protected abstract <I> AggregateStorage<I> getStorage(Class<? extends I> idClass, Class<? extends Entity> aggregateClass);
```
Comment (dmdashenkov) [inline server/src/test/java/io/spine/server/aggregate/AggregateStorageShould.java]: Please check the 100-symbol limit.

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

**Pull request merged**